packages/data-context/src/data/ProjectLifecycleManager.ts

```diff
@@ -230,16 +250,6 @@ export class ProjectLifecycleManager {
 
         await this.setInitialActiveBrowser()
 
-        if (this._currentTestingType && finalConfig.specPattern) {
```

Comment:
How come we don't need this anymore?

Reply:
We still have it, I just moved it up to the top of the function such that we have the specs available for when we spawn the dev-server